controllers/iochaos/fs/types.go

```diff
@@ -84,7 +84,7 @@ func (r *Reconciler) Apply(ctx context.Context, req ctrl.Request, chaos reconcil
 		return err
 	}
 
-	pods, err := utils.SelectAndGeneratePods(ctx, r.Client, &iochaos.Spec)
+	pods, err := utils.SelectAndFilterPods(ctx, r.Client, &iochaos.Spec, iochaos.ObjectMeta)
```

Comment:
Is it okay to just pass namespace? Passing objectMeta seems odd.

Reply:
I agreed.

Reply:
I actually also just pass `namespace` before.But I think 'SelectAndFilterPods' is a very important and widely used function.Almost all chaos objects will invoke this function. And We may extend its logic by this [issue](https://github.com/pingcap/chaos-mesh/issues/193) . The best choice is pass the `InnerObject`.  But it will takes more time to refactor.